tools/Google.Cloud.Tools.ReleaseManager/ContainerCommands/CleanCommand.cs

```diff
@@ -25,6 +25,7 @@ namespace Google.Cloud.Tools.ReleaseManager.ContainerCommands;
 /// - repo-root: path to the root of the clone
 /// - api-path: path to the API to clean generated files from, e.g. google/cloud/functions/v2
 ///   This is optional; when omitted, all configured APIs are cleaned.
+///   The special value of "none" means "clean non-API-specific files only".
```

Comment:
For later: Wondering if `api-path` should be required, with a couple of magic values "none" and "all"?

Alternative, with hide that from the surface and have three commands: clean-api, clean-all, clean-common?

Definetely something for another PR though.

Reply:
You read my mind (first option): https://github.com/googleapis/generator/issues/67

Reply:
Nice :)